ui/uielement.cpp

```diff
@@ -0,0 +1,33 @@
+#include "uielement.h"
+
+#include "pong.h"
+
+namespace pong
+{
+
+void UIElement::Render() const
+{
+}
```

Comment:
Empty methods are normally a little bit of a code smell. It may be better here to force implementation with a pure virtual than accidentally not implement this for a subclass

Reply:
Yeah that makes sense. Im not entirely sure of all the UIElements I need yet, but they probably need to implement render anyway